src/components/Blog/PostLayout.tsx

```diff
@@ -73,6 +73,10 @@ export const PostLayout: FunctionComponent<PostComponentProps> = ({
                 <div className={`blog-post__html ${contentClassName}`}>
                     <MDXRemote {...content} components={components as Components} />
                 </div>
+                <div className="try-sourcegraph">
+                    <h4>Interested in seeing a demo of Sourcegraph?</h4>
+                    <p><a href="https://about.sourcegraph.com/demo">You can request a demo here.</a></p>
+                </div>
```

Comment:
We actually have a `TrySourcegraph` common component that we could use here which includes a relative Link rather than an absolute link to the demo page. I think that would be preferred for design consistency, code-reuse, and the included tracking set on the link. Would that be suitable? You can see this section on our [Use Cases page](https://about.sourcegraph.com/use-cases) right before the footer.